### PR TITLE
Register New User

### LINKED ISSUE #32: Create User

### PR DESCRIPTION
**Pull request by shreygupta2809**
Create a new user
Fixes #32

### CODE REVIEW

**Comment by gsc2001** [comment]
Fixes #32

File: frontend/.eslintcache [hunk 1]
```diff
@@ -1 +1 @@
-[{"/app/src/index.tsx":"1","/app/src/reportWebVitals.ts":"2","/app/src/App.tsx":"3"},{"size":500,"mtime":1608362891091,"results":"4","hashOfConfig":"5"},{"size":425,"mtime":1608362891095,"results":"6","hashOfConfig":"5"},{"size":671,"mtime":1608366205319,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},"1xls6ze",{"filePath":"11","messages":"12","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"10"},{"filePath":"13","messages":"14","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/app/src/index.tsx",[],["15","16"],"/app/src/reportWebVitals.ts",[],"/app/src/App.tsx",[],{"ruleId":"17","replacedBy":"18"},{"ruleId":"19","replacedBy":"20"},"no-native-reassign",["21"],"no-negated-in-lhs",["22"],"no-global-assign","no-unsafe-negation"]
```
Comment (gsc2001) [inline frontend/.eslintcache]: add this file to .gitignore

**Review by gsc2001 — CHANGES_REQUESTED** [review]

File: frontend/.eslintcache (hunk 1 above)
Comment (tichnas) [reply frontend/.eslintcache]: arre, iske me bhi aa gyi ye file :sweat_smile:

File: .gitignore [hunk 2]
```diff
@@ -106,4 +106,6 @@ venv.bak/
 #vscode
 .vscode/
 
-report_unittest.xml
\ No newline at end of file
+report_unittest.xml
+
+e-commerce/
```
Comment (tichnas) [inline .gitignore]: Why this in gitignore?

Comment (shreygupta2809) [reply .gitignore]: i created my venv in E-commerce folder so Gurkirat told me to add it in this
e-commerce is just the name of venv

File: frontend/.eslintcache (hunk 1 above)
Comment (shreygupta2809) [reply frontend/.eslintcache]: Done

**Review by tichnas — APPROVED** [review]

**Review by gsc2001 — APPROVED** [review]

**Pull request merged**